src/common.h

```diff
@@ -1,16 +1,23 @@
 #ifndef COMMON_H
 #define COMMON_H
+
 #include <stdint.h>
 
-#define APFS_MOUNT_DEFAULT_SNAPSHOT         0x0
-#define APFS_MOUNT_LIVEFS                   0x1
-#define APFS_MOUNT_CUSTOM_SNAPSHOT          0x2
-#define APFS_IMG4_SNAPSHOT                  0x3
+
+#define CHECK(expression)            \
```

Comment:
i feel like this macro doesn't have much of a purpose as it's only used once
maybe just put it directly into the function